portfolio/src/main/webapp/index.html

```diff
@@ -38,6 +38,11 @@ <h3>Cheesy Quotes </h3>
      <p>Click to get a quote!</p>
      <button onclick="getQuote()"> Get a Quote </button>
      <div id="quote-container"></div>
+
+     <h3>Sentient </h3>
+     <p>They speak...</p>
+     <button onclick="getMessage"> Listen </button>
```

Comment:
Actionable: `getMessage` needs method parentheses -- `getMessage()`.

Less actionable, more handwavy comment:
Right now, `getMessage()` and `getQuote()` are doing largely the same thing (the only difference is text vs json). If you want to have both, messages and quotes under different buttons, consider splitting your `DataServlet` controller into two controllers -- `MessageDataServlet` and `QuoteDataServlet`.

Reply:
I will definitely split the two later